project/src/main/world/creature/creature-curve.gd

```diff
@@ -21,10 +24,18 @@ export (bool) var _save_curve: bool setget save_curve
 export (Array) var curve_defs: Array setget set_curve_defs
 
 # if false, the curve is made invisible when the creature faces away from the camera.
-export (bool) var visible_when_facing_north: bool = true
+export (bool) var drawn_when_facing_north: bool = true
+
+# if false, the curve is made invisible when the creature faces toward the camera.
```

Comment:
Maybe these comments shouldn't say "visible". Something more like, "The body will not render this curve when the creature faces toward the camera"

Reply:
Changed to `if false, the body will not render this curve when the creature faces away from the camera.`